### ISSUE TITLE
Integrate Kaiko into WDL pipeline

### ISSUE THREAD

**Issue by ssarrafan** [microbiomedata/metaPro#39]
[Kaiko](https://github.com/microbiomedata/kaiko_metaproteome) will be integrated into the metaproteomics WDL pipeline. 

TODO
- [x] Determine fixed and variable input elements that will need to be captured in input.json
- [x] #59
- [ ] #60
- [x] #61
- [x] #62 
- [ ]  Wire Kaiko into metaproteomics pipeline

**Comment by picowatt** [comment]
There's an issue with how we're going to expose the static Kaiko files with the Kaiko container. Alicia pointed me towards a mechanism for doing just that by altering the Cromwell config, but this may require some changes to how Kaiko locates files needed during runtime.

**Comment by ssarrafan** [comment]
Moving to the next sprint per the last comment and the unchecked boxes above.

**Comment by ssarrafan** [comment]
@picowatt @pdpiehowski @CamiloPosso is this still active?

**Comment by pdpiehowski** [comment]
Yes, this issue overlapped somewhat with a backlogged issue, and so I closed that one to clean things up a bit.

**Comment by ssarrafan** [comment]
Not updated in the last 2 weeks. Removing from sprint and adding backlog label.
